### PR TITLE
Models

### PR DESCRIPTION
**Pull request by edwind99**
Adding models

### CODE REVIEW

**Comment by edwinduarte02** [comment]
.

File: credits/admin.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+from django.contrib import admin
+from .models import *
+
+# Register your models here.
+admin.site.register(User)
```
Comment (jorlugaqui) [inline credits/admin.py:5]: `major`: Admins should be registered using their corresponding admins.py files.

File: credits/models.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
+from django.db import models
+
+# Create your models here.
+class User(models.Model):
```
Comment (jorlugaqui) [inline credits/models.py:4]: `major`: Use abstract class instead https://testdriven.io/blog/django-custom-user-model/

**Review by jorlugaqui — APPROVED** [review]
3.0

**Review by jorlugaqui — APPROVED** [review]
3.0

**Pull request merged**